pom.xml

```diff
@@ -89,11 +89,12 @@
     <excludedGroups />
     <memoryMb>4096</memoryMb>
     <directMemoryMb>4096</directMemoryMb>
-    <doclint>none</doclint>
+    <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
Please put this parameter [here](https://github.com/apache/drill/pull/1574/files#diff-600376dffeb79835ede4a0b285078036R907)

Reply:
It makes sense. Done